frontend/src/layout/navigation-3000/sidepanel/panels/sidePanelSettingsLogic.tsx

```diff
@@ -43,7 +43,7 @@ export const sidePanelSettingsLogic = kea<sidePanelSettingsLogicType>([
 
     listeners(({ actions, values }) => ({
         openSettingsPanel: ({ settingsLogicProps }) => {
-            if (!values.featureFlags[FEATURE_FLAGS.POSTHOG_3000]) {
+            if (values.featureFlags[FEATURE_FLAGS.POSTHOG_3000] === 'control') {
```

Comment:
Should this rather be !== "test" ? Thinking that if this returns nothing as it isn't in the experiment then it would evaluate to true 🤔 

(Assume this comment for any other places where we check control)

Reply:
Good catch, it really should be for self-hosted users.